docs-website/versioned_docs/version-1.0.2/replicate/aws.md

```diff
@@ -162,7 +162,9 @@ eksctl utils write-kubeconfig --name $YOUR_EKS_NAME --region eu-west-1
 *Some of the following information was taken from [this link](https://docs.nats.io/running-a-nats-service/nats-kubernetes/nats-external-nlb)
 
 1. One-line installer creates a secure cluster named 'nats’
```

Comment:
Here, I am not sure why the one-line installer is used. I think a helm deployment with a yaml configuration is much better. We can leave this as-is for now but once the new deployment mechanism is finalized it should be updated.